src/core/lib/surface/completion_queue.cc

```diff
@@ -814,6 +814,11 @@ static void cq_end_op_for_pluck(grpc_completion_queue* cq, void* tag,
   GRPC_ERROR_UNREF(error);
 }
 
+static void functor_callback(void* arg, grpc_error* error) {
+  auto* functor = static_cast<grpc_experimental_completion_queue_functor*>(arg);
+  (*functor).functor_run(callback, error == GRPC_ERROR_NONE ? true : false);
```

Comment:
You should drop the entire `? true : false`. I also prefer `functor->functor_run` but am not religious about that.

Reply:
Done and done..